Packages/com.llealloo.audiolink/Runtime/Scripts/GlobalToggle.cs

```diff
@@ -29,13 +29,17 @@ private void Start()
 
         public override void OnDeserialization()
         {
+            if (!enabled) return;
```

Comment:
I guess this is fine, but out of curiosity, which issue were hitting without this?

Reply:
The `if !enabled` on Toggle and Slider Update is for handling a state where Unity still calls the function with the behavior disabled, also added to OnDeserialization just to be sure, never actually checked to see if it caused a difference